core/simulations/helpers/promptToGetNumber.ts

```diff
@@ -0,0 +1,16 @@
+import prompts from 'prompts';
+
+const promptToGetNumber = async (title: string, initial: number) => {
+    const { number } = await prompts([
+        {
+            type: 'number',
+            name: 'number',
+            message: title,
+            initial,
+            min: 1,
```

Comment:
If initial is set (which I assume is selected via just pressing enter), is there a way to set the latest block (without checking it online and typing in?). Eg, by typing `current` or `-1` (https://docs.ethers.org/v5/api/providers/types/#providers-BlockTag)?

Reply:
i don't get the question here.

the latest block is already pre-computed and sent as `initial` arg to this funciton. If the latest block is "100500", then there will be a graytext saying "100500" and one will not have to do anything.

Reply:
This is in case no specific block is suggested, right? But how to enter "latest block" in case of the suggested `14052140` (above in your code)?

Reply:
did not find a simple & straightforward solution. Adjusted the code so that it suggests the latest block instead of the specific block with an active auction (since we have the simulation now that creates auctions, this was redundant anyway)

Reply:
Nit: I guess it's not a requirement now, but as suggested you can accept `-1` input as the latest block (also min can be 0 actually)

Reply:
- adjusted the min to be 0
- i don't think allowing `-1` as latest block is any better than suggesting one by default: at least if the implementation that is currently present is not flawed / bad, there is no benefit (afai can see) in ux/ui.